### PR TITLE
Initial transfer

### PR DESCRIPTION
**Pull request by logan-markewich**

### CODE REVIEW

File: models/gated_scnn/gated_shape_cnn/__init__.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
+from PIL import ImageColor
```
Comment (Lambert-Shirzad) [inline models/gated_scnn/gated_shape_cnn/__init__.py]: @logan-markewich This init file still has the same issue as before.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+The repo contains empty "dad" and "publaynet" folders.
+
+#### DAD
+To setup DAD, run the following starting from the repo root dir:
+
+```
+mkdir dad/documents
+mkdir dad/annotations
+cd ..
+git clone https://github.com/logan-markewich/DAD-Dense-Article-Dataset
```
Comment (Lambert-Shirzad) [inline README.md]: Use the LST repo.

Comment (Lambert-Shirzad) [reply README.md]: @logan-markewich

**Comment by logan-markewich** [comment]
Removed the dead code in an __init__ file

**Review by Lambert-Shirzad — CHANGES_REQUESTED** [review]
See the following comments

**Pull request merged**